FWCore/ParameterSet/python/Modules.py

```diff
@@ -648,12 +695,57 @@ def _assignSwitchProducer():
     )
 )
 """)
+
+            sp.setCase_("test1")
+            self.assertEqual(sp.dumpPython(),
+"""SwitchProducerTest(
+    test1 = cms.EDProducer("Bar",
+        aa = cms.int32(11),
+        bb = cms.PSet(
+            cc = cms.int32(12)
+        )
+    ),
+    test2 = cms.EDProducer("Foo",
+        a = cms.int32(1),
+        b = cms.PSet(
+            c = cms.int32(2)
+        )
+    )
+).setCase_('test1')
```

Comment:
Dumping the "set case" like this looks a bit weird (`setCase_()` returns `self` so it should work), but doing it along
```py
foo = SwitchProducerTest(
...
)
foo.setCase_(...)
```
would require knowing the module label (which is not available here), or extending the functionality of `specialImportRegistry` (would need to think more what exactly that would imply).

I also considered
```py
SwitchProducerTest(
    test1 = ...,
    test2 = ....,
    setCase_ = cms.untracked.string('test1')
)
```
but that appeared to require adding special cases for `setCase_` in many places complicating the implementation even more. Also given that this setting should(?) be somewhat special case, and that it is not passed on to C++, making it a function call felt better choice than a "real parameter".